### PR TITLE
Fixed DocumentItem overflow

### PR DESCRIPTION
**Pull request by Harineko0**
## Summary by CodeRabbit

- **Enhancements**
  - Improved the presentation of document contents by wrapping them in a styled `<p>` tag for better readability.

### CODE REVIEW

File: one-pager-maker/src/stories/DocumentItem.tsx [hunk 1]
```diff
@@ -12,8 +12,10 @@ export const DocumentItem = ({document, onDelete}: {
         <div className={'flex flex-col gap-3'}>
             <Link to={`/edit/${document.owner_id}/${document.id}`}>
                 <div className={'bg-card drop-shadow aspect-[3/4] p-4 rounded-md text-[0.6rem] text-gray-600 ' +
-                    'transition hover:bg-gray-50 text-left whitespace-pre'}>
-                    {document.contents}
+                    'transition hover:bg-gray-50 text-left'}>
+                    <p className={'aspect-[3/4] whitespace-pre-wrap overflow-hidden'}>
```
Comment (binnmti) [inline one-pager-maker/src/stories/DocumentItem.tsx:16]: The important thing is overflow-hidden, right?
What is whitespace-pre-wrap?

**Review by binnmti — APPROVED** [review]
LGTM

File: one-pager-maker/src/stories/DocumentItem.tsx (hunk 1 above)
Comment (Harineko0) [reply one-pager-maker/src/stories/DocumentItem.tsx:16]: This https://zenn.dev/azunasu/articles/ae2b946971ab30

Comment (binnmti) [reply one-pager-maker/src/stories/DocumentItem.tsx:16]: Thank you.
I use whitespace-pre-wrap

**Pull request merged**